### ISSUE TITLE
Binary Search

### ISSUE THREAD

**Issue by SamareshMaiti16** [SamareshMaiti16/PythonDSA#12]
Write the code to perform Binary Search in python.
Guidelines:

- Write the codes in python only.
- Refrain from any unfair means or plagiarism.
- The code should display authenticity.
- Use proper formatting and comments.

**Comment by Sohom2002** [comment]
@SamareshMaiti16, nice of you to add new issues regarding Searching Algorithm. Binary Search always fascinated me from the first day when it was taught in our college. How simple is the approach and yet very much efficient. I would like to contribute in the Binary Search Algo.

**Comment by SamareshMaiti16** [comment]
@Sohom2002 Yes it is fascinating indeed. I am very much grateful for your contribution. I am assigning the work to you. All the best.
